src/Native/Runtime/CMakeLists.txt

```diff
@@ -143,14 +145,19 @@ list(APPEND RUNTIME_SOURCES_ARCH_ASM
   ${ARCH_SOURCES_DIR}/CallDescrWorker.${ASM_SUFFIX}
   ${ARCH_SOURCES_DIR}/CallingConventionConverterHelpers.${ASM_SUFFIX}
   ${ARCH_SOURCES_DIR}/ExceptionHandling.${ASM_SUFFIX}
-  ${ARCH_SOURCES_DIR}/Interlocked.${ASM_SUFFIX}
```

Comment:
It would be better to create empty `Interlocked.asm` file for x86. It will be needed to make things actually work.